tests/ignite/contrib/metrics/test_precision_recall_curve.py

```diff
@@ -138,20 +138,18 @@ def test_check_compute_fn():
 
 
 def _test_distrib_compute(device):
-
     rank = idist.get_rank()
-    torch.manual_seed(12)
+    n_iters = 80
+    batch_size = 16
```

Comment:
I do not like this way to introduce some global constants that are partially used and even overridden, e.g. batch_size. Let's keep previous code: `n_iters = y.shape[0] // batch_size + 1` and `(torch.randint(0, 2, size=(50,)), torch.randint(0, 2, size=(50,)), 16),`

Reply:
Let me correct and resend :)